app/ledger.js

```diff
@@ -981,8 +986,12 @@ var stickyP = (publisher) => {
   var result = (siteSetting) && (siteSetting.get('ledgerPayments'))
 
   // NB: legacy clean-up
-  if ((typeof result === 'undefined') && (typeof synopsis.publishers[publisher].options.stickyP !== 'undefined')) {
-    result = synopsis.publishers[publisher].options.stickyP
+  if (typeof result === 'undefined') {
+    if (typeof synopsis.publishers[publisher].options.stickyP !== 'undefined') {
+      result = synopsis.publishers[publisher].options.stickyP
+    } else {
+      result = true
+    }
     appActions.changeSiteSetting(pattern, 'ledgerPayments', result)
   }
   delete synopsis.publishers[publisher].options.stickyP
```

Comment:
No this is something that needs to be fixed. I have implemented this, but it's causing a crash for some X reason (you noticed this in your first comment). So I removed it for now and will try to find a solution today